### DISCUSSION TITLE
Asus vivobook ryzen 7 5800h

### DISCUSSION THREAD

**Discussion by Mukunya** [DavidS95/Smokeless_UMAF#60] [Q&A]
I have an ASUS Vivobook m3502q with a rysen 7 5800h.

I need to increase the vram, so I booted up this tool and changed the UMA buffer to Uma specified and 2g. I pressed escape until it asked me to save, I said yes.
After reseting the laptop, these settings are back to their defaults.

The goal is to make my and hackintosh run smoother and to eliminate the flashing and graphics bugs caused by having only 512 mb of vram. I know the vram performance will be terrible but it is necessary just to render the internal display and an external 4k monitor.

**Comment by murat-cevik** [comment]
Hi, have you solved this?

**Reply by Mukunya** [reply]
Unfortunately no

**Comment by Ranwiesiel** [comment]
Did you solve it now? XD
